docs/content/spec/auth/scope.md

```diff
@@ -100,7 +100,7 @@ resourcename            := [ hostname '/' ] component [ '/' component ]*
 hostname                := hostcomponent ['.' hostcomponent]* [':' port-number]
 hostcomponent           := /([a-zA-Z0-9]|[a-zA-Z0-9][a-zA-Z0-9-]*[a-zA-Z0-9])/
 port-number             := /[0-9]+/
-action                  := /[a-z]*/
+action                  := /[a-z*]*/
```

Comment:
Wondering if a more accurate presentation would be `[a-z]*` OR `*`, because the suggested change here would also consider `p*u****l` to be valid, and I'm not sure if that was the intent here.